scripts/instagram_automation.py

```
@@ -0,0 +1,229 @@
+        try:
+            response = self.openai_client.chat.completions.create(
+                model="gpt-4",
+                messages=[{"role": "user", "content": prompt}],
+                max_tokens=150,
+                temperature=0.7
+            )
+            
+            caption = response.choices[0].message.content.strip()
+            
+            # Add link in bio CTA
+            caption += f"\n\n🔗 Free large-print puzzles in bio!"
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)